geoportal/geoportailv3_geoportal/static-ngeo/js/OfflineRestorer.js

```diff
@@ -56,11 +56,15 @@ const OfflineRestorer = class extends Restorer {
         .then(() => {
 
           return super.restore(map)
-            .then((extent) => {
-               // Deactivate legacy v3 - bgLayer is handled in v4
-              const bgLayer = this.ngeoBackgroundLayerMgr_.get(map);
-              const mapBoxLayer = bgLayer && bgLayer.getMapBoxMap ? bgLayer : null;
-              this.appMapBoxOffline_.restore(mapBoxLayer);
+            .then(async (extent) => {
+              // Deactivate legacy v3 - bgLayer is handled in v4
+              // await new Promise((r, s) => setTimeout(r, 2000))
```

Comment:
Maybe remove this comment

`// await new Promise((r, s) => setTimeout(r, 2000))`